compiler/circle-mpqsolver/src/core/TestHelper.test.cpp

```diff
@@ -121,6 +121,22 @@ void writeDataToFile(const std::string &path, const std::string &data)
   file.close();
 }
 
+std::string makeTemporaryFolder(char *name_template)
```

Comment:
No. Not ok. It will crash. But `makeTemporaryFile` at line 109 https://github.com/Samsung/ONE/blob/dcf2b6db030ff2b47e83aa4acd2f85fdb60113b5/compiler/circle-mpqsolver/src/core/TestHelper.test.cpp#L109 uses the same pattern. Do i have to fix it (throw an exception or something else)?

Reply:
Hmm.. that code may need to be updated later, but not in this PR.